### PR TITLE
[Spec::Edit] Use Executable.which

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

**Comment by neonichu** [comment]
:ok_hand:

File: lib/cocoapods/command/spec/edit.rb [hunk 1]
```diff
@@ -48,32 +48,23 @@ def run
...
-          return 'vim' if which 'vim'
+          editors = [
+            # Find Sublime Text 2
+            'subl',
+            # Find Textmate
+            'mate',
+            # Find # BBEdit / TextWrangler
+            'edit',
+            # Default to vim
+            'vim',
```
Comment (orta) [inline lib/cocoapods/command/spec/edit.rb]: atom?

Comment (segiddins) [inline lib/cocoapods/command/spec/edit.rb]: wasn't in there originally by I can add it. Where in the list should it go?

Comment (orta) [inline lib/cocoapods/command/spec/edit.rb]: Before vim IMO

Comment (neonichu) [inline lib/cocoapods/command/spec/edit.rb]: vim should obviously be at the top, :trollface:

**Pull request merged**